bin/workflows/pycbc_make_bank_verifier_workflow

```diff
@@ -179,24 +190,51 @@ inp_bank.description='TEMPLATEBANK'
 inp_bank.ifo_list = workflow.ifos
 inp_bank.segment = workflow.analysis_time
 
+# Inspinj Executable
+inspinj_exe = select_generic_executable(workflow, 'injection')
+# The output must be in xml format for pycbc_splitinj to work
+inspinj_exe.extension = ".xml"
+# The following line and the previous one are redundant for
+# lalapps_inspinj, but not for pycbc_create_injections
+inspinj_exe.current_retention_level = wf.Executable.FINAL_RESULT
+
 # Inspinj job
-inspinj_job = LalappsInspinjExecutable(workflow.cp, 'injection', out_dir='.',
-                                       ifos='HL', tags=[])
+inspinj_job = inspinj_exe(workflow.cp, 'injection', out_dir='.',
+                          ifos=workflow.ifos, tags=[])
 
 def add_banksim_set(workflow, file_tag, num_injs, curr_tags, split_banks):
     """Add a group of jobs that does a complete banksim.
     """
-    inspinj_job.update_current_tags(curr_tags)
     t_seg = segments.segment([1000000000, 1000000000+int(num_injs)])
-    node = inspinj_job.create_node(t_seg)
+    if 'LalappsInspinjExecutable' in str(inspinj_exe):
+        inspinj_job.update_current_tags(curr_tags)
+        node = inspinj_job.create_node(t_seg)
+        inj_file = node.output_file
+    elif 'PycbcCreateInjectionsExecutable' in str(inspinj_exe):
+        subsection_name = ('-'.join(['injection']+curr_tags))
+        inj_conf_file_url = workflow.cp.get(subsection_name, 'config-file') 
```

Comment:
Don't do it like this. Add `--config-file` to the list of "file arguments" in PycbcCreateInjectionsExecutable, and then this all happens automatically!

Reply:
This would mean that you don't get to say "leave this file in this directory", but I think that's okay to reduce complexity here.

Reply:
... ALthough I think what you are doing here is adding the injection config files into the output directory, which then might be needed. But we *do* need to set `--config-file` in the list of file arguments for `PycbcCreateInjectionsExecutable` for this to be used in other workflows as well. In that case the config-file argument would always be a local path at this point, but you could still copy it around in this way.